### DISCUSSION TITLE
looking for an IKEv1 config

### DISCUSSION THREAD

**Discussion by gityms123** [strongswan/strongswan#1061] [Q&A]
Why ios can match this configuration. However, Android has always been unable to match

conn IPsec-Xauth-PSK
    keyexchange=ikev1
    ike=aes256-sha256-modp2048,3des-sha1-modp2048,aes256-sha1-modp2048!
    esp=aes256-sha256,3des-sha1,aes256-sha1!
    left=%defaultroute
    leftauth=psk
    leftsendcert=always
    leftsubnet=0.0.0.0/0
    right=%any
    rightauth=psk
    rightauth2=xauth-generic
    rightsourceip=10.31.1.0/24 
    rightsubnet=0.0.0.0/0
    rightsendcert=never
    auto=add

May 24 19:57:37 10[MGR] <1> checkin and destroy IKE_SA (unnamed)[1]
May 24 19:57:37 10[IKE] <1> IKE_SA (unnamed)[1] state change: CONNECTING => DESTROYING
May 24 19:57:37 10[MGR] checkin and destroy of IKE_SA successful

**Comment by tobiasbrunner** [comment]
That log excerpt is obviously not useful at all (see #196). But on iOS and Android you should use IKEv2 anyway (either via built-in client or, on Android, the strongSwan app).

**Reply by gityms123** [reply]
Thanks, but I want to connect via vpn through system settings instead of strongSwan client, and mobile phones below Android 12 can only connect through ikev1, is there any good solution?

**Reply by tobiasbrunner** [reply]
I'm sure it's possible to use IKEv1.
